doc/problem.rst

```diff
@@ -310,5 +310,14 @@ file. It is worth taking a look at the `whole file
    The ``_read_data()`` is not strictly required and is acting as a helper
    function in the code above.
 
+8. If the problem.py file becomes too long and you would like to refactor it, 
```

Comment:
@martin1tab this needs to be formatted correctly: use double backticks for ``problem.py`` (` ``problem.py`` `). Same for the name of the folders ``external_imports`` and ``ramp_kit_dir``, ``ramp-test``, ``sys.path``. We also need a good format for the example, making sure we have the correct indentation.